src/main/io/osd.h

```diff
@@ -54,17 +52,23 @@ typedef struct {
     uint16_t alt_alarm;
 
     uint8_t video_system;
-    osd_unit_t units;
-} osd_profile_t;
+    uint8_t row_shiftdown;
```

Comment:
This is equivalent to SHIFTDOWN in MWOSD; shifts down entire drawing area by specified lines (0 or 1 in reality). Comes in handy when you want to use the same layout on multiple monitors with different responses to VSYNC.